chain/client/src/chunk_validation.rs

```diff
@@ -646,7 +728,35 @@ impl Client {
     ) -> Result<(), Error> {
         let shard_id = chunk.shard_id();
         let chunk_hash = chunk.chunk_hash();
-        let Some(witness) = self.create_state_witness(prev_chunk_header, chunk)? else {
+        let chunk_header = chunk.cloned_header();
+
+        let transactions_validation_storage_config = RuntimeStorageConfig {
+            state_root: chunk_header.prev_state_root(),
+            use_flat_storage: true,
+            source: StorageDataSource::Db,
+            state_patch: Default::default(),
+            record_storage: true,
+        };
+
+        let Ok(validated_transactions) = validate_prepared_transactions(
```

Comment:
could you please add comment here explaining why do we call `validate_prepared_transaction` here